### ISSUE TITLE
Colab notebooks issue

### ISSUE THREAD

**Issue by juan190199** [google/neural-tangents#187]
Dear All,

I believe there is some problem when importing neural-tangents. When I ran the code in colab, I got the message below.

Thank you,
Juan

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
[<ipython-input-4-8ba94ffea59f>](https://1qleeixthu2-496ff2e9c6d22116-0-colab.googleusercontent.com/outputframe.html?vrz=colab-20230728-060123-RC00_551806953#) in <cell line: 11>()
      9 import tensorflow_datasets as tfds
     10 
---> 11 import neural_tangents as nt
     12 from neural_tangents import stax

5 frames
[/usr/local/lib/python3.10/dist-packages/neural_tangents/__init__.py](https://1qleeixthu2-496ff2e9c6d22116-0-colab.googleusercontent.com/outputframe.html?vrz=colab-20230728-060123-RC00_551806953#) in <module>
     19 __version__ = '0.6.3'
     20 
---> 21 from . import experimental
     22 from . import predict
     23 from . import stax

[/usr/local/lib/python3.10/dist-packages/neural_tangents/experimental/__init__.py](https://1qleeixthu2-496ff2e9c6d22116-0-colab.googleusercontent.com/outputframe.html?vrz=colab-20230728-060123-RC00_551806953#) in <module>
     13 # limitations under the License.
     14 
---> 15 from .empirical_tf.empirical import empirical_ntk_fn_tf
     16 from .empirical_tf.empirical import get_apply_fn_and_params

[/usr/local/lib/python3.10/dist-packages/neural_tangents/experimental/empirical_tf/empirical.py](https://1qleeixthu2-496ff2e9c6d22116-0-colab.googleusercontent.com/outputframe.html?vrz=colab-20230728-060123-RC00_551806953#) in <module>
     95 
     96 from jax.experimental import jax2tf
---> 97 from neural_tangents._src.empirical import NtkImplementation, empirical_ntk_fn, DEFAULT_NTK_IMPLEMENTATION, _DEFAULT_NTK_FWD, _DEFAULT_NTK_J_RULES, _DEFAULT_NTK_S_RULES
     98 from neural_tangents._src.utils.typing import Axes, PyTree, VMapAxes
     99 import tensorflow as tf

[/usr/local/lib/python3.10/dist-packages/neural_tangents/_src/empirical.py](https://1qleeixthu2-496ff2e9c6d22116-0-colab.googleusercontent.com/outputframe.html?vrz=colab-20230728-060123-RC00_551806953#) in <module>
    120 from jax.util import safe_map as map, safe_zip as zip
    121 import numpy as onp
--> 122 from .utils import rules
    123 from .utils import utils
    124 from .utils.typing import ApplyFn, Axes, EmpiricalGetKernelFn, EmpiricalKernelFn, PyTree, VMapAxes, VMapAxisTriple

[/usr/local/lib/python3.10/dist-packages/neural_tangents/_src/utils/rules.py](https://1qleeixthu2-496ff2e9c6d22116-0-colab.googleusercontent.com/outputframe.html?vrz=colab-20230728-060123-RC00_551806953#) in <module>
   1046   return np.zeros(cts_in.shape + invals[idx].shape, cts_in.dtype)  # pytype: disable=unsupported-operands  # always-use-return-annotations
   1047 
-> 1048 STRUCTURE_RULES[jax.ad.zeros_like_p] = _eye_s
   1049 JACOBIAN_RULES[jax.ad.zeros_like_p] = _zeros_like_j
   1050 

[/usr/local/lib/python3.10/dist-packages/jax/_src/deprecations.py](https://1qleeixthu2-496ff2e9c6d22116-0-colab.googleusercontent.com/outputframe.html?vrz=colab-20230728-060123-RC00_551806953#) in getattr(name)
     51       warnings.warn(message, DeprecationWarning, stacklevel=2)
     52       return fn
---> 53     raise AttributeError(f"module {module!r} has no attribute {name!r}")
     54 
     55   return getattr

AttributeError: module 'jax' has no attribute 'ad'
```

**Comment by romanngg** [comment]
Thanks for pointing this out and sorry for late reply -- this shoudl be fixed in https://github.com/google/neural-tangents/commit/4de699a899faf9f429c721a2512b7b68347f45b3, please use the version from head for now, and I will soon release a new v0.6.3 to pypi with these changes.

**Comment by romanngg** [comment]
Fixed in [v0.6.4](https://github.com/google/neural-tangents/releases/tag/v0.6.4), thanks noticing!
